server/src/main/resources/org/opensearch/bootstrap/test-framework.policy

```diff
@@ -114,6 +114,7 @@ grant codeBase "${codebase.opensearch-core}" {
 grant codeBase "${codebase.opensearch-nio}" {
   // opensearch-nio makes and accepts socket connections
   permission java.net.SocketPermission "*", "accept,resolve,connect";
+  permission java.io.FilePermission "${java.io.tmpdir}${/}-", "delete";
```

Comment:
Should this be full permission to tmp, not just delete?

Reply:
It does not look like we need that,  `BootstrapForTesting` should cover it: https://github.com/opensearch-project/OpenSearch/blob/main/test/framework/src/main/java/org/opensearch/bootstrap/BootstrapForTesting.java#L150